st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -58,6 +73,37 @@ def get_all(self, **kwargs):
     def get_one(self, ref_or_id):
         return super(ActionAliasController, self)._get_one(ref_or_id)
 
+    @jsexpose(arg_types=[str], status_code=http_client.ACCEPTED)
+    def match(self, command, **kwargs):
```

Comment:
Can user also pass other arguments to this function? If not, we should probably get rid of `**kwargs`.